tests/complex_diagnostics.ts

```diff
@@ -0,0 +1,25 @@
+interface Value<T> {
```

Comment:
This file doesn't seem to be called from anywhere?

Reply:
I was using it for testing, but forgot to flesh out the whole test.  I might just replace the TypeScript error test with this as it generates a full structure in TypeScript (2 diagnostics, including one diagnostics with a diagnostic chain and related information).